cron/getIncrementBalance.js

```diff
@@ -0,0 +1,29 @@
+const cron = require('node-cron');
+const { familyModel } = require('../database/models');
+
+const { getLogger } = require('../helpers');
+const logger = getLogger('getIncrementBalance');
+
+async function main() {
+  try {
+    await cron.schedule('1 * * * *', async function () {
+
+      const families = await familyModel.find({});
+
+      families.forEach((item) => {
+        item.balance = item.totalSalary + item.passiveIncome;
+        item.giftsForUnpacked = item.balance / item.flatSquareMeters;
```

Comment:
Are you sure that this field called `giftsForUnpacked`? Also result would be a floating point number, which is not what we expect to see in that field